CHANGES.md

```diff
@@ -1,5 +1,26 @@
 # Changes
 
+## edge-21.5.2
+
+This edge release updates the proxy-init container to check whether the iptables
+rules have already been added, which prevents errors if the proxy-init container
+is restarted. Also, the linkerd cli now has tab completion for kubernetes
+resources, saving you precious keystrokes! Finally, the proxy has been updated
```

Comment:
Only `viz stat` command has resource completion. There is a change up for all `viz` commands to have resource completion. After that, the other extensions and core still will not have resource completion. I would change this to mention that `viz stat` has it, and more is on the way.